community-meetings/2022-01-07.md

```diff
@@ -0,0 +1,18 @@
+# Agenda for the GoReleaser Community Meeting - January 7, 2022
```

Comment:
What time will the meeting be? I'd love to join 😄

Reply:
@caarlos0: as we are in different time zones: What do you suggest?

Reply:
ping @caarlos0

Reply:
I just checked the members of the org and we are located from Seattle UTC-8 to Moscow UTC+3 => Let's take UTC-2 :-)

Reply:
The time idk yet as well... since its a friday, it would have to be after hours to me (so after 7pm likely).

My suggestion is to maybe change it to the saturday, then with some effort anyone can join (without jobs in the way at least)... thoughts?

Reply:
not sure, maybe we should postpone it to end of January. Don't know if the 7 is now to short in notice?

Reply:
yeah, agreed

I'm also already pretty much swamped in the beginning of the year :(